### ISSUE TITLE
a11y improvements for Platform overview and Help

### ISSUE THREAD

**Issue by maria-jon** [openaq/openaq.org#1129]
Platform overview and Help pages accessibility can improve in a few places: 

## Platform overview

- Multiple h1 tags

## Help 
- Missing aria-label on graphic icon
- Missing focus state styling on tabs
- Three links without text content:  
  - 2 x `<a href="https://www.epa.gov/air-sensor-toolbox"></a>` 
  - `<a href="https://www.c40knowledgehub.org/s/article/Making-the-invisible-visible-A-guide-for-mapping-hyperlocal-air-pollution-to-drive-clean-air-action?language=en\_US"></a>`
